packages/portfolio-planner/src/entrypoint.js

```diff
@@ -1,26 +1,33 @@
 #! /usr/bin/env node
+/* global globalThis */
```

Comment:
Since this is a Node script,
```suggestion
/* eslint-env node */
```

Reply:
Done.